index.html

```diff
@@ -985,7 +987,7 @@ <h3>Types</h3>
         <p>
 Software systems that process the kinds of objects specified in this document
 use type information to determine whether or not a provided <a>credential</a>
-or <a>presentation</a> is appropriate. Type information SHOULD be expressed
+or <a>presentation</a> is appropriate. Type information MUST be expressed
```

Comment:
Similar to my previous comment, I think that as written, this may be difficult/impossible to test.
There is a separation of concerns, one is what properties a verifiable credential that adheres to the data model MUST contain, the other is how the properties SHOULD be used.
The first is clearly testable, the second is not. I feel we are being too squishy in our data model if we conflate the conformance requirements of the two concerns.
A verifiable credential MUST include a type property. Type information SHOULD be expressed using the type property.

Reply:
Fixed, I think? By your PR.